configure.ac

```diff
@@ -116,12 +116,12 @@ AC_DEFUN([CH_CHECK_VERSION], [
       [AC_MSG_RESULT([ok ($vact)])
        AC_SUBST([$1_VERSION], [$vact])],
       [AC_MSG_RESULT([too old ($vact)])
+	   AC_SUBST([$1_VERSION_OLD], ["too old $vact"]),
```

Comment:
I like this approach, but let's take it a little further to simplify the report. In the current PR, report lines look like:

```
sphinx-build(1) ≥ $vmin_sphinx ... ${SPHINX_VERSION:-not found $SPHINX_VERSION_OLD} ${SPHINX}
```

Let's instead do this:

```
sphinx-build(1) ≥ $vmin_sphinx ... ${SPHINX_VERSION_NOTE}
```

That is, `CH_CHECK_VERSION` sets `FOO_VERSION_NOTE` to one of three values:

* If `FOO` is not found (i.e., `test -n "$prog"` fails), then set `FOO_VERSION_NOTE` to "not found".
* If `FOO` is found and version is OK, then set `FOO_VERSION_NOTE` to "ok (x.y.z)".
* If `FOO` is found and too old, then set `FOO_VERSION_NOTE` to "too old (x.y.z)".

`FOO_VERSION` goes away.

Reply:
I really like that approach. I'll implement it.